pages/projects/[id]/edit.js

```diff
@@ -0,0 +1,38 @@
+import Form from "@/components/Form";
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import { mutate } from "swr";
+
+export default function EditPage() {
+  const router = useRouter();
+
+  const { id } = router.query;
```

Comment:
the id can be undefined in an edgecase, namely if the editpage is opened directly and not via a Link in the App. Then the first render will return an empty query object and only after a second render the id will be available. Therefore, please check if the id is actually there and if not return a loading message to be rendered instead of the page.

Reply:
we solved it like this now:

  if (!project || isLoading || !id) {
    return <p>is Loading...</p>;
  }